.github/workflows/action-lint-actions.yaml

```diff
@@ -0,0 +1,19 @@
+name: reviewdog
+on: [pull_request]
+jobs:
+  actionlint-actions:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      - name: Build actionlint flags
+        id: build_flags
+        run: |
+          files="$(find actions -type f \( -name '*.yml' -o -name '*.yaml' \) | paste -sd' ' -)"
```

Comment:
It is better to search for the name "action" with one of these formats. It can contain other YAML files that are not actions

Reply:
What do you mean? `yaml` files can have both file extensions, so it's best to look for both.

Reply:
Changed.